content/general/limits.textile

```diff
@@ -0,0 +1,306 @@
+
+Any clients that attempt to join the presence set over the presence member limit will be rejected and the error code @91003@ will be returned.
+
+h3(#channel-create). Channel creation rate
+
+The channel creation rate is the number of channels that can be created per second.
+
+The limit is calculated based on the hard limit for "peak channels":#peak-channels, with a minimum value of 50 per second for self-service and business accounts.
+
+If the channel creation rate is exceeded, an "instantaneous rate limit":#instantaneous-rates will be applied to channel creation.
+
+h3(#message-publish-rate). Message publish rate
+
+The message rate is the maximum rate at which messages can be published on a channel.
+
+If the message publish rate is exceeded for a channel, an "instantaneous rate limit":#instantaneous-rates on publishing will be applied to that channel.
+
+h2(#message). Message limits
+
+Message limits relate to the number, rate and bandwidth of "messages":/realtime/messages on your account.
+
+|_=. Limit |_=. Free |_=. Self-service |_=. Business |_=. Enterprise |
+| "Message size":#message-size (KiB) | 16 | 64 | 64 | 256 |
+| "History TTL":#history-ttl (hours) | 24 | 72 | 72 | Custom |
+
+h3(#message-size). Message size
+
+The message size is the maximum size of a single published "message":/glossary#messages.
+
+Note that the average message size is set to 2KiB. Although a message can be published that is 16KiB, this will count as 8 messages towards your "message count":#messages.
+
+Attempting to publish a message in excess of the message size limit will fail and thd the error code @40009@ will be returned.
+
+h3(#history-ttl). History TTL
+
+History time to live (TTL) is the maximum time that a message or presence event can be retrieved from "history":/realtime/history.
+
+h2(#queues). Queue limits
+
+Queue limits relate to the number, length and rates of "queues":/glossary#queues.
+
+|_=. Limit |_=. Free |_=. Self-service |_=. Business |_=. Enterprise |
+| "Number of queues":#queue-numbers (per account) | 5 | 50 | 50 | Custom |
+| "Queue length":#queue-length (per account) | 10,000 | 50,000 | 50,000 | Custom |
+| "Queue publish rate":queue-publish (per second) | 100 | 200 | 200 | Custom |
+| "Queue TTL":#queue-ttl (hours) | 1 | 24 | 24 | Custom |
+
+h3(#queue-numbers). Number of queues
+
+The number of queues is the maximum number of queues that can be created. This limit includes all applications in your account.
+
+h3(#queue-length). Queue length
+
+Queue length is the maximum number of messages that can be stored in queues whilst waiting to be consumed. This value is shared between all queues on the account.
+
+For example, with a free account you could have one queue with a length of 10,000 messages, or two queues each with a length of 5,000 messages.
+
+h3(#queue-publish). Queue publish rate
+
+The queue publish rate is the maximum rate that messages can be published to a queue at, per second.
```

Comment:
..."the maximum rate at which messages can be published to a queue" ?

Reply:
Updated.